src/pudl/settings.py

```diff
@@ -647,6 +648,76 @@ class EtlSettings(BaseSettings):
     pudl_in: str = pudl.workspace.setup.get_defaults()["pudl_in"]
     pudl_out: str = pudl.workspace.setup.get_defaults()["pudl_out"]
 
+    @root_validator(pre=False)
+    def raw_table_validation_ferc1(cls, field_values):
+        """Require the presence of raw FERC1 tables needed for PUDL table creation."""
+        # only check if we are actually loading any pudl tables
+        if field_values["datasets"] and field_values["ferc_to_sqlite_settings"]:
```

Comment:
I know we typically try to avoid having multiple return statements, but one common exception to that rule is preemptively bailing out of a function when you don't have to do anything, rather than enclosing the whole thing in a conditional. All else being equal I think having less indentation and less code inside the conditionals helps with readability.

Reply:
sure! yeah in this case there is a clear "bail early" option